### ISSUE TITLE
Single entry in array not shown in xls

### ISSUE THREAD

**Issue by DebbautSara** [optilude/xlsx-template#71]
Situation:  I need to show one or multiple data entries in an xls.
When I have 2 or more entries, everything is perfect, xls gets rendered with the correct headers in the columns.  When I have 1 entry, only the headers are generated in the xls. 

Current workarround: When the array.length == 1, add an empty entry to the array. In the xls only one entry is shown because the second entry is empty and we can't really check if it's there.
`if (filteredEvents.length == 1) {
     this.addEmptyEvent(filteredEvents);
}`

Please help.

**Comment by Sedjj** [comment]
I tried to examine the source code and found the condition that leads to this bug.

```
if(placeholder.full && placeholder.type === "table" && substitution instanceof Array) {
	newCellsInserted = self.substituteTable(
		row, newTableRows,
		cells, cell,
		namedTables, substitution, placeholder.key
	);

	// don't double-insert cells
	if (newCellsInserted !== 0 
		|| substitution.length <= 1) {
		appendCell = false;
	}

	// Did we insert new columns (array values)?
	if(newCellsInserted !== 0) {
		cellsInserted += newCellsInserted;
		self.pushRight(self.workbook, sheet.root, cell.attrib.r, newCellsInserted);
	}
}
```
here it must be replaced by `substitution.length <= 0`
```
// don't double-insert cells
if (newCellsInserted !== 0 
	|| substitution.length <= 0) {
	appendCell = false;
}
```
I do not know how this will affect the rest of the functionality, but it is necessary that the developer looked!!

**Comment by kant2002** [comment]
Released in 0.3.0 Please verify that this is solved your issues
